scripts/docker_build.sh

```diff
@@ -45,127 +45,123 @@ image_build() {
     shift
 
     docker buildx build \
-        -f "$GIT_ROOT/Dockerfile" \
         --target "$target" \
         --tag "${DOCKER_IMAGE_ROOT}/$name:$version" \
         --tag "${DOCKER_IMAGE_ROOT}/$name:latest" \
         "$@" .
 }
 
+# Minimimal ORT image
```

Comment:
So first there are complain that i made a big commit with al parts that change, and i should split for better understand.
Now i should squash all commits again because there are too many commits and mostly return to the initial situation ?